### PR TITLE
feat(data-exploration): implement values on series

### PR DESCRIPTION
**Pull request by thmsobrmlr**
## Problem

The value on series filter isn't implemented for data exploration yet.

## Changes

This PR adds the filter to data exploration

## How did you test this code?

Verifying the filter works by default for the pie chart and that it can be set / unset.

### CODE REVIEW

File: cypress/e2e/insights.dataExploration.cy.ts [hunk 1]
```diff
@@ -65,7 +65,7 @@ describe('Insights (with data exploration on)', () => {
             cy.get('.funnels-empty-state__title').should('exist')
         })
 
-        it('can open a new retention insight', () => {
+        it.skip('can open a new retention insight', () => {
```
Comment (Twixes) [inline cypress/e2e/insights.dataExploration.cy.ts:68]: What's the plan for this?

File: plugin-server/src/config/config.ts [hunk 2]
```diff
@@ -74,6 +74,7 @@ export function getDefaultConfig(): PluginsServerConfig {
         JOB_QUEUE_GRAPHILE_URL: '',
         JOB_QUEUE_GRAPHILE_SCHEMA: 'graphile_worker',
         JOB_QUEUE_GRAPHILE_PREPARED_STATEMENTS: false,
+        JOB_QUEUE_GRAPHILE_CONCURRENCY: 1,
```
Comment (Twixes) [inline plugin-server/src/config/config.ts:77]: This seems completely unrelated

File: plugin-server/src/types.ts [hunk 3]
```diff
@@ -132,6 +132,7 @@ export interface PluginsServerConfig {
     JOB_QUEUE_GRAPHILE_URL: string // use a different postgres connection in the graphile worker
     JOB_QUEUE_GRAPHILE_SCHEMA: string // the postgres schema that the graphile worker
     JOB_QUEUE_GRAPHILE_PREPARED_STATEMENTS: boolean // enable this to increase job queue throughput if not using pgbouncer
+    JOB_QUEUE_GRAPHILE_CONCURRENCY: number // concurrent jobs per pod
```
Comment (Twixes) [inline plugin-server/src/types.ts:135]: And this

**Review by Twixes — CHANGES_REQUESTED** [review]
Frontend code looks good but I think some changes are unrelated?

**Comment by thmsobrmlr** [comment]
Sorry, must have been juggling too many branches @Twixes. Rebase with master resolved the unrelated changes.

**Review by Twixes — APPROVED** [review]
